controllers/imageclusterinstall_controller.go

```diff
@@ -186,49 +202,72 @@ func (r *ClusterConfigReconciler) setHostConfiguredCondition(ctx context.Context
 		cond.Message = err.Error()
 	}
 
-	patch := client.MergeFrom(config.DeepCopy())
-	meta.SetStatusCondition(&config.Status.Conditions, cond)
-	return r.Status().Patch(ctx, config, patch)
+	patch := client.MergeFrom(clusterInstall.DeepCopy())
+	meta.SetStatusCondition(&clusterInstall.Status.ConfigConditions, cond)
+	return r.Status().Patch(ctx, clusterInstall, patch)
 }
 
-func (r *ClusterConfigReconciler) mapBMHToCC(ctx context.Context, obj client.Object) []reconcile.Request {
+func (r *ImageClusterInstallReconciler) mapBMHToICI(ctx context.Context, obj client.Object) []reconcile.Request {
 	bmh := &bmh_v1alpha1.BareMetalHost{}
 	bmhName := obj.GetName()
 	bmhNamespace := obj.GetNamespace()
 
 	if err := r.Get(ctx, types.NamespacedName{Name: bmhName, Namespace: bmhNamespace}, bmh); err != nil {
 		return []reconcile.Request{}
 	}
-	ccList := &relocationv1alpha1.ClusterConfigList{}
-	if err := r.List(ctx, ccList); err != nil {
+	iciList := &relocationv1alpha1.ImageClusterInstallList{}
```

Comment:
Not for this PR, but I think we want some annotation on the BMH that say the BMH is pre-installed with template SNO